public/index.html

```diff
@@ -68,6 +69,44 @@
           ]
         }
       }
+      var ASPerformanceViews = {
+        'GB/1/174': {
+          id: 'GB/1/174',
+          as: 174,
+          risk: 1,
+          country: 'GB',
+          type: 'as/performance',
+          isFetched: false,
+          isFetching: false,
+          didFailed: false,
+          measure: "count_normalized",
+          selectorConfig: [
+            {disabled: true, as: 174},
+            {disabled: false, as: undefined},
```

Comment:
@zelima we have discussed this on Friday standup and decided to not have countrywide data in graph for now. Instead we have the main AS + 4 editable dropdowns for adding other ASs.